onnxruntime/core/mlas/inc/mlas.h

```diff
@@ -1762,6 +1762,7 @@ MLASCALL
 MlasConvDepthwise(
     const MLAS_FP16* const* Input,
     const MLAS_FP16* Filter,
+    const MLAS_FP16* Bias,
```

Comment:
please add documentation for the new parameter

Reply:
Added